src/Xamarin.Android.Tools.AndroidSdk/Tests/AndroidSdkInfoTests.cs

```diff
@@ -64,6 +64,36 @@ public void Constructor_Paths ()
 			}
 		}
 
+		[Test]
+		public void Ndk_PathInSdk()
+		{
+			if (!OS.IsWindows)
+				Assert.Ignore("This only works in Windows");
```

Comment:
Android Studio, placed the NDK into `$HOME/Library/Android/sdk/ndk-bundle`.

Unix needs to be supported as well.

Reply:
The thing is I know how Windows is looking for the NDK (in VS) but I'm not sure what Mac and Unix are doing, so I didn't want to write potentially breaking code. Mac and Unix are not modified right now, and they are in separate classes so they really don't need to. This comment is so the test will always work. There are other tests here doing the opposite (excluding Windows), so it may be common usage to do that here?

Reply:
If mac & Unix are that complicated to figure out, we need to simplify things. :-)

There is a macOS PR builders, so it _is_ possible to ensure that things work there.

Reply:
I don't think the problem is complexity. The thing is that this logic applies to Windows because of what we do in VS, but I think the existing logic will work for mac just fine, and Unix doesn't have an IDE constraint here. The change is in a class that's specialized to Windows, and the tests are very generic (and that's why I'm ignoring it for non-Windows). There are other tests in this same class that are ignoring Windows explicitly for the same reason.

Reply:
@sgmunn will probably know if the logic may apply to mac, based on what VSMac is expecting.

Reply:
@bratsche should have more recent knowledge on this than I do right now.